frontend/src/routes/_index/route.tsx

```diff
@@ -26,6 +26,7 @@ import store, { RootState } from "#/store";
 import { removeFile, setInitialQuery } from "#/state/initial-query-slice";
 import { clientLoader as rootClientLoader } from "#/root";
 import { UploadedFilePreview } from "./uploaded-file-preview";
+import config from "#/../public/config.json";
```

Comment:
I don't think this will work. The `import` happens at build time,  not at runtime. So we can't replace the config after the dockerfile has been built.

Instead, we need to `fetch` the JSON file